### PR TITLE
doc(release): prepare release notes for Centreon Web 22.10.3

### PR DESCRIPTION
**Pull request by tuntoja**
## Description

Prepare release notes for Centreon Web 22.10.3

## Target version

- [ ] 21.10.x (staging)
- [ ] 22.04.x (staging)
- [x] 22.10.x (staging)
- [ ] Cloud (staging)
- [ ] Plugin Packs (staging)
- [ ] 23.04.x (next)

### CODE REVIEW

File: i18n/fr/docusaurus-plugin-content-docs/version-22.10/releases/centreon-os.md [hunk 1]
```diff
@@ -18,6 +18,49 @@ Retrouvez plus de détails sur la version 22.10 dans notre [post de blog](https:
...
+
+Release date: `soon`
+
+#### Enhancements
+
+- [APIv2] Added vault configuration endpoint
+
+#### Bug fixes
+
+- Fixed a bug that blocked exporting multiple pollers' configuration at he same time with a shared severity
```
Comment (cg-tw) [inline i18n/fr/docusaurus-plugin-content-docs/version-22.10/releases/centreon-os.md:31]:
```suggestion
- Fixed a bug that blocked exporting multiple pollers' configuration at the same time with a shared severity
```

File: i18n/fr/docusaurus-plugin-content-docs/version-22.10/releases/centreon-os.md [hunk 2]
```diff
@@ -18,6 +18,49 @@ Retrouvez plus de détails sur la version 22.10 dans notre [post de blog](https:
...
+- [Authentication] Fixed authentication with complete url for token endpoint
```
Comment (cg-tw) [inline i18n/fr/docusaurus-plugin-content-docs/version-22.10/releases/centreon-os.md:32]:
```suggestion
- [Authentication] Fixed authentication with complete URL for token endpoint
```

File: i18n/fr/docusaurus-plugin-content-docs/version-22.10/releases/centreon-os.md [hunk 3]
```diff
@@ -18,6 +18,49 @@ Retrouvez plus de détails sur la version 22.10 dans notre [post de blog](https:
...
+- Fixed bug whre it's impossible to filter on hosts and/or services in 22.10.
```
Comment (cg-tw) [inline i18n/fr/docusaurus-plugin-content-docs/version-22.10/releases/centreon-os.md:33]:
```suggestion
- Fixed bug where it's impossible to filter on hosts and/or services in Centreon 22.10.
```

File: i18n/fr/docusaurus-plugin-content-docs/version-22.10/releases/centreon-os.md [hunk 4]
```diff
@@ -18,6 +18,49 @@ Retrouvez plus de détails sur la version 22.10 dans notre [post de blog](https:
...
+- Fixed an ACL bug allowing users to modify users even when they had read-only permissions
+- Acknowledgement/downtimes failed without returning an error when the character case for the resource was wrong
+- Fixed a bug linked to forbidden characters in Engine object names that could strip the '0', '3' and '9' digits from host names
+- Allow users to configure Broker's global maximum retention
+- Fixed an issue that caused months to be missing from the calendar selection
+- Bugfix of pagination on event log page
```
Comment (cg-tw) [inline i18n/fr/docusaurus-plugin-content-docs/version-22.10/releases/centreon-os.md:39]:
```suggestion
- Fixed bug with pagination on event log page
```

File: i18n/fr/docusaurus-plugin-content-docs/version-22.10/releases/centreon-os.md [hunk 5]
```diff
@@ -18,6 +18,49 @@ Retrouvez plus de détails sur la version 22.10 dans notre [post de blog](https:
...
+- Recurrent downtime editing issues with disabled objects
```
Comment (cg-tw) [inline i18n/fr/docusaurus-plugin-content-docs/version-22.10/releases/centreon-os.md:40]:
```suggestion
- Fixed recurrent downtime editing issues with disabled objects
```

File: i18n/fr/docusaurus-plugin-content-docs/version-22.10/releases/centreon-os.md [hunk 6]
```diff
@@ -18,6 +18,49 @@ Retrouvez plus de détails sur la version 22.10 dans notre [post de blog](https:
...
+- [Resource-Status] Monitoring engine command displayed in detail regarding ACL configured
+- Fix view contact notfications window
```
Comment (cg-tw) [inline i18n/fr/docusaurus-plugin-content-docs/version-22.10/releases/centreon-os.md:42]:
```suggestion
- Fixed "view contact notifications" window
```

File: i18n/fr/docusaurus-plugin-content-docs/version-22.10/releases/centreon-os.md [hunk 7]
```diff
@@ -18,6 +18,49 @@ Retrouvez plus de détails sur la version 22.10 dans notre [post de blog](https:
...
+- [Authentication] Fixed authentication with complete url for token endpoint
+- Fixed bug whre it's impossible to filter on hosts and/or services in 22.10.
+- Fixed an ACL bug allowing users to modify users even when they had read-only permissions
+- Acknowledgement/downtimes failed without returning an error when the character case for the resource was wrong
+- Fixed a bug linked to forbidden characters in Engine object names that could strip the '0', '3' and '9' digits from host names
+- Allow users to configure Broker's global maximum retention
+- Fixed an issue that caused months to be missing from the calendar selection
+- Bugfix of pagination on event log page
+- Recurrent downtime editing issues with disabled objects
+- [Resource-Status] Monitoring engine command displayed in detail regarding ACL configured
```
Comment (cg-tw) [inline i18n/fr/docusaurus-plugin-content-docs/version-22.10/releases/centreon-os.md:41]: what does the sentence mean?

File: i18n/fr/docusaurus-plugin-content-docs/version-22.10/releases/centreon-os.md [hunk 8]
```diff
@@ -18,6 +18,49 @@ Retrouvez plus de détails sur la version 22.10 dans notre [post de blog](https:
...
+- Fix view contact notfications window
+- [API] Fixed API access when user doesn't have access to UI
+- Fixed issue pagination not displayed in all legacy pages
```
Comment (cg-tw) [inline i18n/fr/docusaurus-plugin-content-docs/version-22.10/releases/centreon-os.md:44]:
```suggestion
- Fixed an issue where the pagination was not displayed in all legacy pages
```

File: i18n/fr/docusaurus-plugin-content-docs/version-22.10/releases/centreon-os.md [hunk 9]
```diff
@@ -18,6 +18,49 @@ Retrouvez plus de détails sur la version 22.10 dans notre [post de blog](https:
...
+
+#### Security fixes
+
+- [Core] Fixed vulnerabilities in functions.js file
+- [Configuration] Sanitized queries in the list of services by host group
+- [Configuration] Sanitized queries in the list of host categories
+- [Configuration] Sanitized queries in the list of commands
+- [Core] Fixed vulnerabilities in ajaxLdapSearch.js file
+- [Configuration] Sanitized queries in the list of broker configurations
```
Comment (cg-tw) [inline i18n/fr/docusaurus-plugin-content-docs/version-22.10/releases/centreon-os.md:53]:
```suggestion
- [Configuration] Sanitized queries in the list of Broker configurations
```

File: i18n/fr/docusaurus-plugin-content-docs/version-22.10/releases/centreon-os.md [hunk 10]
```diff
@@ -18,6 +18,49 @@ Retrouvez plus de détails sur la version 22.10 dans notre [post de blog](https:
...
+- [Core] Fixed vulnerabilities in color_picker.php
+- [Core] Fixed vulnerabilities in pathway.php
+- [Core] Fixed vulnerabilities in color_picker_mb.php
+- [Core] Fixed vulnerabilities in rename.php
+- [Configuration] Fixed vulnerabilities in services listing
+- [Configuration] Fixed vulnerabilities in host form
+- [API] Fixed API access when user doesn't have access to UI
+- Fixed issue pagination not displayed in all legacy pages
```
Comment (cg-tw) [inline i18n/fr/docusaurus-plugin-content-docs/version-22.10/releases/centreon-os.md:61]: already appears above

**Review by cg-tw — CHANGES_REQUESTED** [review]

File: versioned_docs/version-22.10/releases/centreon-os.md [hunk 11]
```diff
@@ -19,6 +19,48 @@ Read more about version 22.10 in our [blog post](https://www.centreon.com/en/blo
...
+- [Configuration] Sanitized queries in the list of commands
+- [Core] Fixed vulnerabilities in ajaxLdapSearch.js file
+- [Configuration] Sanitized queries in the list of Broker configurations
+- [Core] Fixed vulnerabilities in color_picker.php
+- [Core] Fixed vulnerabilities in pathway.php
+- [Core] Fixed vulnerabilities in color_picker_mb.php
+- [Core] Fixed vulnerabilities in rename.php
+- [Configuration] Fixed vulnerabilities in services listing
+- [Configuration] Fixed vulnerabilities in host form
+- Fixed issue pagination not displayed in all legacy pages
```
Comment (mdebauchez) [inline versioned_docs/version-22.10/releases/centreon-os.md:61]: This line is already in the Bug fixes part (line 45)

File: i18n/fr/docusaurus-plugin-content-docs/version-22.10/releases/centreon-os.md [hunk 12]
```diff
@@ -18,6 +18,48 @@ Retrouvez plus de détails sur la version 22.10 dans notre [post de blog](https:
...
+- [Configuration] Sanitized queries in the list of commands
+- [Core] Fixed vulnerabilities in ajaxLdapSearch.js file
+- [Configuration] Sanitized queries in the list of Broker configurations
+- [Core] Fixed vulnerabilities in color_picker.php
+- [Core] Fixed vulnerabilities in pathway.php
+- [Core] Fixed vulnerabilities in color_picker_mb.php
+- [Core] Fixed vulnerabilities in rename.php
+- [Configuration] Fixed vulnerabilities in services listing
+- [Configuration] Fixed vulnerabilities in host form
+- Fixed issue pagination not displayed in all legacy pages
```
Comment (mdebauchez) [inline i18n/fr/docusaurus-plugin-content-docs/version-22.10/releases/centreon-os.md:60]: This line is already in the Bug fixes part (line 44)

**Review by mdebauchez — CHANGES_REQUESTED** [review]

**Review by mdebauchez — APPROVED** [review]

File: i18n/fr/docusaurus-plugin-content-docs/version-22.10/releases/centreon-os.md [hunk 13]
```diff
@@ -18,6 +18,47 @@ Retrouvez plus de détails sur la version 22.10 dans notre [post de blog](https:
...
+- [Authentication] Fixed authentication with complete URL for token endpoint
+- Fixed bug where it's impossible to filter on hosts and/or services in Centreon 22.10.
+- Fixed an ACL bug allowing users to modify users even when they had read-only permissions
+- Acknowledgement/downtimes failed without returning an error when the character case for the resource was wrong
+- Fixed a bug linked to forbidden characters in Engine object names that could strip the '0', '3' and '9' digits from host names
+- Allow users to configure Broker's global maximum retention
+- Fixed an issue that caused months to be missing from the calendar selection
+- Fixed bug with pagination on event log page
+- Fixed recurrent downtime editing issues with disabled objects
+- [Resource-Status] Fixed an issue causing ACL action "Display executed command by monitoring engine" to not be applied on the Resource Status page
```
Comment (cg-tw) [inline i18n/fr/docusaurus-plugin-content-docs/version-22.10/releases/centreon-os.md:41]:
```suggestion
- [Resources Status] Fixed an issue causing ACL action "Display executed command by monitoring engine" to not be applied on the Resources Status page
```

File: versioned_docs/version-22.10/releases/centreon-os.md [hunk 14]
```diff
@@ -19,6 +19,47 @@ Read more about version 22.10 in our [blog post](https://www.centreon.com/en/blo
...
+- [Authentication] Fixed authentication with complete URL for token endpoint
+- Fixed bug where it's impossible to filter on hosts and/or services in Centreon 22.10.
+- Fixed an ACL bug allowing users to modify users even when they had read-only permissions
+- Acknowledgement/downtimes failed without returning an error when the character case for the resource was wrong
+- Fixed a bug linked to forbidden characters in Engine object names that could strip the '0', '3' and '9' digits from host names
+- Allow users to configure Broker's global maximum retention
+- Fixed an issue that caused months to be missing from the calendar selection
+- Fixed bug with pagination on event log page
+- Fixed recurrent downtime editing issues with disabled objects
+- [Resource-Status] Fixed an issue causing ACL action "Display executed command by monitoring engine" to not be applied on the Resource Status page
```
Comment (cg-tw) [inline versioned_docs/version-22.10/releases/centreon-os.md:42]:
```suggestion
- [Resources Status] Fixed an issue causing ACL action "Display executed command by monitoring engine" to not be applied on the Resources Status page
```

**Review by cg-tw — APPROVED** [review]

**Review by cg-tw — DISMISSED** [review]

**Review by mdebauchez — DISMISSED** [review]

File: i18n/fr/docusaurus-plugin-content-docs/version-22.10/releases/centreon-os.md [hunk 15]
```diff
@@ -18,6 +18,48 @@ Retrouvez plus de détails sur la version 22.10 dans notre [post de blog](https:
 
 ## Centreon Web
 
+### 22.10.3
+
+Release date: `January 2023, 9`
```
Comment (cg-tw) [inline i18n/fr/docusaurus-plugin-content-docs/version-22.10/releases/centreon-os.md:23]:
```suggestion
Release date: `January 9, 2023`
```

File: versioned_docs/version-22.10/releases/centreon-os.md [hunk 16]
```diff
@@ -19,6 +19,48 @@ Read more about version 22.10 in our [blog post](https://www.centreon.com/en/blo
 
 ## Centreon Web
 
+### 22.10.3
+
+Release date: `January 2023, 9`
```
Comment (cg-tw) [inline versioned_docs/version-22.10/releases/centreon-os.md:24]:
```suggestion
Release date: `January 9, 2023`
```

**Review by cg-tw — CHANGES_REQUESTED** [review]

**Review by cg-tw — APPROVED** [review]

**Review by mdebauchez — APPROVED** [review]

File: versioned_docs/version-22.10/releases/centreon-os.md [hunk 17]
```diff
@@ -19,6 +19,48 @@ Read more about version 22.10 in our [blog post](https://www.centreon.com/en/blo
 
 ## Centreon Web
 
+### 22.10.3
+
+Release date: `January 9, 2023`
+
```
Comment (cg-tw) [inline versioned_docs/version-22.10/releases/centreon-os.md:25]:
```suggestion

> There is currently a known issue affecting version 22.10.3. Please do not update your platform. A new version will be released shortly.

```

File: i18n/fr/docusaurus-plugin-content-docs/version-22.10/releases/centreon-os.md [hunk 18]
```diff
@@ -18,6 +18,50 @@ Retrouvez plus de détails sur la version 22.10 dans notre [post de blog](https:
 
 ## Centreon Web
 
+### 22.10.3
+
+Release date: `January 9, 2023`
+
+Un problème connu affecte actuellement la version 22.10.3. Patientez avant de mettre votre plateforme à jour : une nouvelle version va être publiée sous peu.
```
Comment (cg-tw) [inline i18n/fr/docusaurus-plugin-content-docs/version-22.10/releases/centreon-os.md:25]:
```suggestion
> Un problème connu affecte actuellement la version 22.10.3. Patientez avant de mettre votre plateforme à jour : une nouvelle version va être publiée sous peu.
```

**Review by cg-tw — APPROVED** [review]

**Review by mdebauchez — APPROVED** [review]

**Pull request merged**